reflex/components/recharts/charts.py

```diff
@@ -522,6 +510,15 @@ def create(cls, *children, **props) -> Component:
             height=props.pop("height", "100%"),
         )
 
+    def get_event_triggers(self) -> dict[str, Union[Var, Any]]:
+        """Get the event triggers that pass the component's value to the handler.
+        Returns:
+            A dict mapping the event trigger to the var that is passed to the handler.
+        """
+        return {
+            EventTriggers.ON_ANIMATION_START: lambda: [],
+            EventTriggers.ON_ANIMATION_START: lambda: [],
```

Comment:
typo with copy/paste? should be END I guess

Reply:
Yes, thank you!